tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -1326,7 +1321,7 @@ class DxcCompiler : public IDxcCompiler3,
       std::string msg("Internal Compiler error: ");
       switch (hr) {
       case DXC_E_VALIDATOR_MISSING:
-        msg = "Error: external validator selected, but DXIL.dll not found.";
+        msg = "Error: DXIL.dll not found.";
```

Comment:
You've removed the only place this is generated, so you can remove this case entirely.  It was only related to external validation, so irrelevant now.